src/iexamine.cpp

```diff
@@ -2282,16 +2282,9 @@ void iexamine::tree_hickory(player &p, const tripoint &examp)
 
 item_location maple_tree_sap_container() {
     return g->inv_map_splice( []( const item &it ) { 
-        return (
-            it.is_bucket() ||
-            it.is_watertight_container()
-            ) && (
-            it.is_container_empty() || (
-                !it.is_container_full( true ) && 
-                it.contents.front().type->id == "maple_sap"
-                )
-            );
-        }, _( "Which container:" ), PICKUP_RANGE );
+        static const item maple_sap = item( "maple_sap", 0 );
```

Comment:
I can not put it in the function body because it will be out of scope for the lambda. I could put it global but that won't make it update on mod load/unload. I'm guessing the best case would be to remove the `static`. What do you think?